Dockerfile

```diff
@@ -1,7 +1,10 @@
-FROM python:3.7
+FROM ubuntu
```

Comment:
1. Agree with sonatype-scanner. Better pin the version exactly here.
2. As I know - *ubuntu* has a bit legacy *gpg* binary (2.2), and we need 2.3 or later because of key algo support improved. Seems better base on *fedora:38* which has the most recent *gpg* and *pysvn*.